test/index.js

```diff
@@ -10,39 +10,39 @@ describe('normalize', function() {
     var type = 'string';
     var value = 'test string';
     var result = normalize(type, value);
-    expect(result).toEqual(value);
+    expect(result).toBe(value);
```

Comment:
I had been wondering about this as well, we definitely want to do strict comparisons here.